src/get-dart.md

```diff
@@ -11,14 +11,14 @@ The Dart SDK has the libraries and command-line tools that you need to develop
 Dart command-line, server, and non-Flutter web apps.
 For details, see the [Dart SDK overview](/tools/sdk).
 
-**As of Flutter 1.20, the [Flutter SDK][flutter] includes the Dart SDK.**
+**As of Flutter 1.21, the [Flutter SDK][flutter] includes the Dart SDK.**
```

Comment:
This should remain 1.20, right?

Reply:
Nope, Flutter 1.20 uses Dart 2.9 which doesn't have it (it did temporarily, but 1.20 stable doesn't)